packages/patternfly-4/react-core/src/helpers/util.ts

```diff
@@ -82,3 +83,41 @@ export function fillTemplate(templateString, templateVars) {
   const func = new Function(...Object.keys(templateVars), `return \`${templateString}\`;`);
   return func(...Object.values(templateVars));
 }
+
+/** This function allows for keyboard navigation through dropdowns. The custom argument is optional.
+ * @param {number} index The index of the element you're on
+ * @param {string} position The orientation of the dropdown
+ * @param {string[]} refsCollection Array of refs to the items in the dropdown
+ * @param {Object[]} kids Array of items in the dropdown
+ * @param {boolean} custom Allows for handling of flexible content
```

Comment:
to indicate that custom is optional this should be :
```suggestion
 * @param {boolean} [custom] Allows for handling of flexible content
```

Reply:
Thanks, I'm new to this. I'll push that now.